### PR TITLE
Cache sys env

### PR DESCRIPTION
**Pull request by jimmy-e**

### CODE REVIEW

File: R/s3path.r [hunk 1]
```diff
@@ -4,10 +4,11 @@ s3path <- function() {
   ## The default S3 prefix, for example, `s3://yourbucket/yourprefix/`.
   ## You should set this in everyone's `~/.Rprofile` if
   ## you are using s3mpi to collaborate in a data science team.
-  path <- getOption("s3mpi.path")
+  ## System environment variables are also accepted.
+  path <- getOption("s3mpi.path", Sys.getenv("s3mpi.path"))
```
Comment (kirillseva) [inline R/s3path.r]: for environment variables the convention would be to do

```r
path <- getOption("s3mpi.path", Sys.getenv("S3MPI_PATH"))
```

that's why I had to write a little helper in this PR https://github.com/kirillseva/mailgunner/pull/3

File: DESCRIPTION [hunk 2]
```diff
@@ -28,4 +28,4 @@ License: MIT
 LazyData: true
 Roxygen: list(wrap = FALSE)
 VignetteBuilder: knitr
-RoxygenNote: 6.0.1
```
Comment (kirillseva) [inline DESCRIPTION]: I think you should update your roxygen version to avoid unnecessary changes

Comment (jimmy-e) [reply DESCRIPTION]: wait, how did roxygen 5.0.0 get in here?
